### PR TITLE
added email func

### PR DESCRIPTION
**Pull request by ismnitesh**

### CODE REVIEW

File: babe [hunk 1]
```diff
@@ -115,18 +156,26 @@ def listCommand(listArgParser):
...
-    #print("Config: " + configFile)
+        data = myfile.read()
     r = requests.post(url("/v2/apps"), data = data, headers = {'content-type': 'application/json'}, auth=httpBasicCreds, verify=httpsVerify)
     if r.status_code == 201:
         print("App deployment request accepted. Use list command to check")
+
+        jsonData = json.loads(data)
+        app_owner_email = None
+        if 'app.owner.email' in jsonData['labels']:
+            app_owner_email = jsonData['labels']['app.owner.email']
```
[the remark this answers is not in the document]
Comment (ismnitesh) [reply babe]: Done.

File: babe [hunk 2]
```diff
@@ -37,6 +41,49 @@ userAuth=False
 httpBasicCreds = None
 protocol = "http"
 httpsVerify = True
+default_config_path="/etc/babe/config.yml"
+babe_config = None
+disable_email = False
+
+def read_config(config_path):
```
Comment (santanusinha) [inline babe]: camelCase everywhere

File: babe [hunk 3]
```diff
@@ -37,6 +41,49 @@ userAuth=False
...
+    try:
+        msg = MIMEMultipart()
+        msg['Subject'] = subject
+        msg['From'] = babe_config['sender_email']
+        if not None == cc:
+            msg['CC'] = cc
+        if None == to:
+            raise Exception('To param cannot be empty for email')
+        msg['To'] = to
+        message_body = MIMEText(content, 'html')
```
Comment (santanusinha) [inline babe]: camelCase

File: babe (hunk 2 above)
Comment (ismnitesh) [reply babe]: done.

File: babe (hunk 3 above)
Comment (ismnitesh) [reply babe]: done.

**Pull request merged**